src/Three.d.ts

```diff
@@ -32,7 +32,6 @@ export * from './textures/CubeTexture';
 export * from './textures/CanvasTexture';
 export * from './textures/DepthTexture';
 export * from './textures/Texture';
-export * from './geometries/Geometries';
```

Comment:
It seems this line is required so the geometry generators are exported.

Reply:
Yeah, not sure what happened there. Will fix.